src/Libraries/CoreNodeModels/Input/File.cs

```diff
@@ -38,6 +38,14 @@ public Filename() : base("Filename")
         {
             ShouldDisplayPreviewCore = false;
         }
+
+        public override bool IsAllowedInCustomizer
+        {
+            get
+            {
+                return false;
+            }
+        }
```

Comment:
Let's remove this field as discussed above.  We'll need to filter disallowed nodes in `DynamoPublish`.